### PR TITLE
Store only build compiled assets on public

### PR DESCRIPTION
**Pull request by joecohens**
//cc @jbrooksuk @GrahamCampbell 

Closes https://github.com/cachethq/Cachet/issues/463 and inline components on https://github.com/cachethq/Cachet/issues/384

### CODE REVIEW

File: src/Models/Incident.php [hunk 1]
```diff
@@ -93,13 +93,13 @@ public function getIconAttribute()
     {
```
Comment (joecohens) [inline src/Models/Incident.php]: This should be moved to the presenter on other PR

Comment (jbrooksuk) [inline src/Models/Incident.php]: Agreed!

**Comment by jbrooksuk** [comment]
Done.

**Comment by jbrooksuk** [comment]
@joecohens I've pulled and this doesn't actually work anymore. JS assets are no longer compiled. Did I break that?

**Comment by joecohens** [comment]
@jbrooksuk Have you reinstalled the npm modules?

**Pull request merged**